packages/core/src/material/UnlitMaterial.ts

```diff
@@ -86,16 +88,42 @@ export class UnlitMaterial extends Material {
         break;
       case AlphaMode.Blend:
         {
+          this.shaderData.disableMacro("ALPHA_CUTOFF");
+
           target.sourceColorBlendFactor = target.sourceAlphaBlendFactor = BlendFactor.SourceAlpha;
           target.destinationColorBlendFactor = target.destinationAlphaBlendFactor = BlendFactor.OneMinusSourceAlpha;
           target.colorBlendOperation = target.alphaBlendOperation = BlendOperation.Add;
           depthState.writeEnabled = false;
           this.renderQueueType = RenderQueueType.Transparent;
         }
         break;
+      case AlphaMode.CutOff:
+        {
+          this.shaderData.enableMacro("ALPHA_CUTOFF");
+
+          target.sourceColorBlendFactor = target.sourceAlphaBlendFactor = BlendFactor.One;
+          target.destinationColorBlendFactor = target.destinationAlphaBlendFactor = BlendFactor.Zero;
+          target.colorBlendOperation = target.alphaBlendOperation = BlendOperation.Add;
+          depthState.writeEnabled = true;
+          this.renderQueueType = RenderQueueType.AlphaTest;
+        }
+        break;
     }
   }
```

Comment:
we want unlit material more powerful.

- alphaCutoff can independent. Whether transparent or not。

We have two design:

1. chang alphaMode to renderMode(opaque, transparent, transparentAddtive).
2. another is more clearly,but need two API, one is RenderMode(opaque, transparent), one is blendMode(alpha, addtive, ...)